i18n/en.json

```diff
@@ -1135,6 +1135,8 @@
   "admin.permissions.group.manage_private_channel_members_and_read_groups.name": "Manage Channel Members",
   "admin.permissions.group.manage_public_channel_members_and_read_groups.description": "Add and remove public channel members.",
   "admin.permissions.group.manage_public_channel_members_and_read_groups.name": "Manage Channel Members",
+  "admin.permissions.group.manage_shared_channels.description": "Manage Shared Channels",
+  "admin.permissions.group.manage_shared_channels.name": "Shared Channels",
```

Comment:
All the other names start with a verb.

Reply:
Makes sense. These are more of dummy names. I am waiting for Katie's recommendations on the verbiage.

Reply:
@agnivade Curious if there are other permissions just allow the creation of shared channels?  If so, wondering if we should be explicit like "Create Shared Channels"

Reply:
I think this is just a single umbrella permission for now that can change a normal channel to a shared channel, and revert it too. @wiggin77 - Is that right? Or did you have something else in mind?

Reply:
@agnivade If its an umbrella permission I am good with "Manage Shared Channels"

Reply:
This permission allows someone to share a channel, invite another cluster to sync with the shared channel, unshare a channel.

Reply:
Cool, I'll go with "Manage Shared Channels" then.

Reply:
@agnivade Martin clarified yesterday how permission scope is designed. Since this permission is not attached to any channel id, since the permission allow creation of any shared channel, then the scope should be system.